src/server/api/routes/usercv.router.js

```diff
@@ -0,0 +1,31 @@
+/* This is the router for cvs */
+
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const usercvController = require('../controllers/usercv.controller');
+
+/**
+ * @swagger
+ * /modules:
+ *  get:
+ *    summary: Get all cvs and users
```

Comment:
Maybe it should say that there is a limit on 20?

Reply:
Sure, Patrick,.let me change on swagger documentation, Thanks